docs: troubleshooting section (#216)
grafana/k6-studio: README.md

---

## Troubleshooting

### `localhost` requests not being recorded

The proxy is not capturing traffic when sent directly to `localhost`, to remedy that you can assign a hostname to it and make requests through that name.

> Modify the `hosts` file on your system for example `127.0.0.1 myapp` and then you will be able to make requests in the browser at `myapp:8000/path`.

Reviewer: ```suggestion
Modify the `hosts` file on your system, for example `127.0.0.1 myapp`, and then you will be able to make requests in the browser at `myapp:8000/path`.
```

> The proxy is not capturing traffic when sent directly to `localhost`, to remedy that you can assign a hostname to it and make requests through that name.
> Modify the `hosts` file on your system for example `127.0.0.1 myapp` and then you will be able to make requests in the browser at `myapp:8000/path`.

Reviewer 2: ```suggestion
The proxy doesn't capture traffic when sent directly to `localhost`. To fix that, you can assign a hostname to it and make requests through that name.
To do that, modify the `hosts` file on your system, for example, `127.0.0.1 myapp`, and then you will be able to make requests in the browser at `myapp:8000/path`.
```
